convert2rhel/checks.py

```diff
@@ -57,6 +62,88 @@ def perform_pre_ponr_checks():
     ensure_compatibility_of_kmods()
 
 
+def check_convert2rhel_latest():
+    """Make sure that we are running the latest version of convert2rhel"""
+
+    # get lastest avalible version
+
+    repo_content = (
+        "[convert2rhel]\n"
+        "name=Convert2RHEL Repository\n"
+        "baseurl=https://ftp.redhat.com/redhat/convert2rhel/$releasever/os/\n"
+        "gpgcheck=0\n"
+        "enabled=1\n"
+    )
+
+    repo_dir = os.path.join(utils.TMP_DIR, "convert2rhel_repo")
+    utils.mkdir_p(repo_dir)
+    # repo_dir = os.path.join("/tmp/","convert2rhel_repo")
+    repo_path = os.path.join(repo_dir, "convert2rhel.repo")
+    store_content_to_file(filename=repo_path, content=repo_content)
+
+    # repoquery --setopt=reposdir=$(pwd)/test_repo --releasever=8 convert2rhel
+    raw_output_convert2rhel_versions, _ = run_subprocess(
+        "repoquery --releasever={releasever} --setopt=reposdir={repodir} convert2rhel".format(
+            releasever=system_info.releasever,
+            # releasever="8",
+            repodir=repo_dir,
+        ),
+        print_output=False,
+    )
```

Comment:
And also, as referenced in the description in the Jira Issue, we should be checking if this fails (maybe with the return_code) because the user might not have an internet connection at the time of running this. 

```python
    raw_output_convert2rhel_versions, return_code = run_subprocess(
        "repoquery --releasever={releasever} --setopt=reposdir={repodir} convert2rhel".format(
            releasever=system_info.releasever,
            # releasever="8",
            repodir=repo_dir,
        ),
        print_output=False,
    )
    
    if return_code != 0: 
        loggerinst.warnin("Couldn't check if this is the latest version of convert2rhel")
```

Reply:
I have added the check to see if this check fails.